### DISCUSSION TITLE
What release do I need?

### DISCUSSION THREAD

**Discussion by scooter024** [hyperion-project/hyperion.ng#1239] [General]
Hi, im new to this and dealing with raspberry pi. I'm not the best when it comes to computers as im more a mechanical type of guy. I was following a vid on YouTube for a hyperion project for my TV. I'm using a raspberry pi zero w and in his vid he said I needed to use arm6 file from the 2.0.0 alpha 6. Now I see that there is a alpha 9 file out. Do I use the alpha 9 file or stay with 6? If I should use 9 do I still use the arm version of that file? Thanks.

**Comment by Paulchen-Panther** [comment]
> Do I use the alpha 9 file or stay with 6?

alpha 9

> If I should use 9 do I still use the arm version of that file?

yes
